classes/plugins/PluginHelper.inc.php

```diff
@@ -240,7 +244,11 @@ function upgradePlugin($category, $plugin, $path, &$errorMsg) {
 
 			$upgradeFile = $pluginDest . '/' . PLUGIN_UPGRADE_FILE;
 			if($fileManager->fileExists($upgradeFile)) {
+                                $request = Application::getRequest();
+                                $site = $request->getSite();
 				$params = $this->_getConnectionParams();
+                                $params['locale'] = $site->getPrimaryLocale();
+                                $params['additionalLocales'] = $site->getSupportedLocales();
```

Comment:
It looks like your editor is using spaces for indents; could you change these to tab-based indents?